sopel/plugins/exceptions.py

```diff
@@ -0,0 +1,12 @@
+# coding=utf-8
+from __future__ import unicode_literals, absolute_import, print_function, division
+
+
+class PluginError(Exception):
+    pass
+
+
+class PluginNotRegistered(PluginError):
```

Comment:
I think this should be:
```suggestion
        message = 'Plugin "%s" not registered' % name
```